pkg/controller/dynamic/dynamic_controller_integration_test.go

```diff
@@ -728,8 +735,12 @@ func assertObjectContains(t *testing.T, obj, changedFields map[string]interface{
 			}
 			assertObjectContains(t, objVal.(map[string]interface{}), changedVal.(map[string]interface{}))
 		default:
-			if !reflect.DeepEqual(objVal, changedVal) {
-				t.Fatalf("unexpected value for %v: got %v, want %v", changedKey, objVal, changedVal)
+			if diff := cmp.Diff(objVal, changedVal, cmpopts.SortSlices(
```

Comment:
This is because of the label randomization I take it?  I don't love the fact that it potentially covers up issues in other resources, but I don't have a great answer.

Reply:
Yeah I figure anyplace where there are slices it would run into some non deterministic ordering.